pos_frequency_pipeline.py

```diff
@@ -2,7 +2,68 @@
 Implementation of POSFrequencyPipeline for score ten only.
 """
 
+import json
+from pathlib import Path
+import re
+
+from pymystem3 import Mystem
+
+from constants import ASSETS_PATH
+from pipeline import CorpusManager
+from visualizer import visualize
+
 
 class POSFrequencyPipeline:
     def __init__(self, assets):
+        self.assets = assets
+        self.text_to_process = ''
+        self.pos_frequencies = {}
+
+    def run(self):
+        """
+        Runs pipeline process scenario
+        """
+        articles = self.assets.get_articles()
+        for file in articles.values():
+            self.text_to_process = file.get_raw_text()
+            self.pos_frequencies.update(self._process())
+            self._write_to_meta(file.article_id)
+            visualize(statistics=self.pos_frequencies,
+                      path_to_save=Path(ASSETS_PATH) / '{}_image.png'.format(file.article_id))
+
+    def _process(self):
+        """
+        Performs processing of each text
+        """
+        mystem_analyser = Mystem()
```

Comment:
the pipeline works on top of the processed dataset and does not perform analysis. Single Responsibility principle (!)